pkgs/flox/default.nix

```
@@ -26,13 +26,14 @@
   pkgsFor,
   floxVersion,
   flox-pkgdb,
+  writeScript,
 }: let
   # crane (<https://crane.dev/>) library for building rust packages
   craneLib = inputs.crane.mkLib pkgsFor;
 
   # build time environment variables
   envs =
-    {
+    rec {
```

```suggestion
    {
```

and use `${envs.ENV_FROM_LOCKFILE_PATH}` below, `rec` can have some funky side effects